scripts/install.sh

```diff
@@ -20,6 +20,11 @@ function bluemix_auth() {
   echo "Installing kubectl"
   chmod +x ./kubectl
   sudo mv ./kubectl /usr/local/bin/kubectl
+
+  echo "Installing OpenWhisk"
+  curl -LO https://openwhisk.ng.bluemix.net/cli/go/download/linux/amd64/wsk
+  chmod +x ./wsk
+  sudo mv ./wsk /usr/local/bin/wsk
```

Comment:
Should this be under `bluemix_auth()`? Would it make more sense to pull this and the installation of `kubectl` from here and make an `install_dependancies()` function?